src/screens/collection/collection.tsx

```diff
@@ -77,34 +77,40 @@ export const Collection = () => {
   );
 
   return (
-    <ScrollView style={styles.root}>
-      <FlatList
-        data={collectibles}
-        renderItem={renderItem}
-        horizontal
-        showsHorizontalScrollIndicator={false}
-        onViewableItemsChanged={handleChanged}
-        removeClippedSubviews={true}
-        snapToInterval={snapToInterval}
-        viewabilityConfig={{
-          itemVisiblePercentThreshold: 50,
-          minimumViewTime: 200
-        }}
-        decelerationRate={0}
-        scrollEventThrottle={16}
-        keyExtractor={keyExtractor}
-        onEndReached={() => {
-          if (collectibles.length % PAGINATION_STEP === 0 && !isLoading) {
-            setOffset(offset + PAGINATION_STEP);
-          }
-        }}
-        onEndReachedThreshold={0.5}
-        ListEmptyComponent={emptyComponent}
-        ListFooterComponent={footerComponent}
-        ListHeaderComponent={<View style={styles.emptyBlock} />}
-        windowSize={3}
-        initialNumToRender={3}
-      />
+    <ScrollView style={styles.root} contentContainerStyle={styles.scrollViewContainer}>
```

Comment:
is this ScrollView necessary? You have a FlatList inside

Reply:
yes, bc we need vertical scroll for some types of devices
(we have horizontal Flatlist inside)

Reply:
ok. did you checked how this works on Android? There could be some issues with gestures

Reply:
yes, I checked it, and QA saw it, it works correctly now

Reply:
ok